api/pkg/cli/cmd/search/testdata/TestSearch_JSONFormat.golden

```diff
@@ -23,6 +24,7 @@
 				"WebURL": "http://web.github.com/foo-bar/",
 				"UpdatedAt": "2020-01-01 12:00:00 +0000 UTC",
 				"Platforms": null,
+				"HubURLPath": null,
```

Comment:
why this is null?

Reply:
So the latest version has a type ResourceVersion data with a view of `withoutResource` where it didn't had the attribute of `hubURLPath` 
I have added it now 👍🏻